frontend/src/app/home/home.component.ts

```diff
@@ -4,10 +4,12 @@ import { AuthService } from '../auth.service';
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
-  styleUrls: ['./home.component.scss']
+  styleUrls: ['./home.component.scss'],
 })
 export class HomeComponent {
-
   constructor(public authService: AuthService) {}
 
+  logIn(): void {
+    window.location.replace('/api/oauth2/google?redirect_url=/app');
```

Comment:
**issue:** the `redirect_url` should be the current path over static `/app`

Reply:
Won't the current path always be '/', since this method is only called from the home page?

Reply:
True dat